stdlib/2and3/pstats.pyi

```diff
@@ -0,0 +1,36 @@
+from profile import Profile
+from cProfile import Profile as cProfile
+from typing import Any, AnyStr, Dict, IO, List, Tuple, TypeVar, Union, overload
+
+_Selector = Union[AnyStr, float, int]
+_T = TypeVar('_T', bound='Stats')
+
+class Stats:
+    # As best I can tell, the first argument has to be a string or Profile
+    # instance and the remaining ones may also be Stats instances
+    # the only keyword argument accepted is "stream"
+    def __init__(self: _T, *args: Union[AnyStr, Profile, _T],
```

Comment:
Instead of `AnyStr`, it looks like this only works with `str`. So maybe replace `AnyStr` with `str`?